.github/ISSUE_TEMPLATE/feature.md

```
@@ -0,0 +1,25 @@
+---
+name: Feature
+about: New capability — player-facing or system
+labels: feature
+---
+
+<!-- Lead paragraph: what ships and why, ≤2 sentences. -->
+
+## Player story
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)